Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -777,12 +777,21 @@ namespace AZ
                     const float approxScreenPercentage =
                         ModelLodUtils::ApproxScreenPercentage(pos, lodData.m_lodSelectionRadius, cameraPos, yScale, isPerspective);
 
-                    for (const Cullable::LodData::Lod& lod : lodData.m_lods)
+                    AssetQuality bias = lodData.m_lodConfiguration.m_lodBias;
+                    for (AZStd::size_t i = 0; i < lodData.m_lods.size(); ++i)
                     {
+                        const Cullable::LodData::Lod& lod = lodData.m_lods[i];
+
                         // Note that this supports overlapping lod ranges (to suport cross-fading lods, for example)
                         if (approxScreenPercentage >= lod.m_screenCoverageMin && approxScreenPercentage <= lod.m_screenCoverageMax)
                         {
-                            addLodToDrawPacket(lod);
+                            AZStd::size_t biasedLod = i + bias;
+                            if (biasedLod >= lodData.m_lods.size())
+                            {
+                                break;
+                            }
```

Comment:
It's not skipping the rendering, but biasing its lod. This check is only there to prevent an out of bound access. The bias value is maxed to lodCount - 2 which guarantees that at least a LOD will be visible.